### PR TITLE
feat(cbr): add policies data source support

### PR DESCRIPTION
**Pull request by jinyangyang222**
**What this PR does / why we need it**:
add policies data source support
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
add policies data source support
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST="./huaweicloud/services/acceptance/cbr" TESTARGS="-run TestAccPoliciesDataSource_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cbr -v -run TestAccPoliciesDataSource_basic -timeout 360m -parallel 4
=== RUN   TestAccPoliciesDataSource_basic
=== PAUSE TestAccPoliciesDataSource_basic
=== CONT  TestAccPoliciesDataSource_basic
--- PASS: TestAccPoliciesDataSource_basic (11.87s)
PASS

```

### CODE REVIEW

File: docs/data-sources/cbr_policies.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  so the default value is **false**.
+
+* `destination_project_id` - The ID of the replication destination project.
+
+* `backup_quantity` - The maximum number of retained backups. The value ranges from `2` to `99,999`.
+  This parameter and `time_period` are alternative.
+
+* `time_period` - The duration (in days) for retained backups. The value ranges from `2` to `99,999`.
+
+-> **NOTE:** If `backup_quantity` and `time_period` are both left blank, the backups will be retained permanently.
```
Comment (ShiChangkuo) [inline docs/data-sources/cbr_policies.md]: I think all notes can be deleted as they are only attributes.

File: docs/data-sources/cbr_policies.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+Use this data source to get available CBR policies within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable policy_id {}
+
+data "huaweicloud_cbr_policies" "test" {
+  policy_id = var.policy_id
```
Comment (ShiChangkuo) [inline docs/data-sources/cbr_policies.md]: use `name` or `type` will be more suitable.

File: docs/data-sources/cbr_policies.md (hunk 1 above)
Comment (jinyangyang222) [reply docs/data-sources/cbr_policies.md]: fixed.

File: docs/data-sources/cbr_policies.md (hunk 2 above)
Comment (jinyangyang222) [reply docs/data-sources/cbr_policies.md]: fixed.

**Pull request merged**